src/Stripe.net/Services/Cards/CardService.cs

```diff
@@ -6,7 +6,7 @@ namespace Stripe
     using System.Threading.Tasks;
     using Stripe.Infrastructure;
 
-    public class CardService : Service<Card>,
+    public class CardService : ServiceNested<Card, Customer>,
```

Comment:
Likely not a problem but flagging since I thought about it. What if we have a generic nested resource in the future (like card and bank account used to be) and we want to have it shared? Would we simply add a parent class/interface or similar instead?